CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ cmake_minimum_required(VERSION 2.6)
 SET (CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/.orogen/config")
 INCLUDE(motor_controllerBase)
 
-# FIND_PACKAGE(KDL)
-# FIND_PACKAGE(OCL)
-
+if (ROCK_TEST_ENABLED)
```

Comment:
I've been wondering about this for a while now... It always seemed to me that `ROCK_TEST_ENABLED` was an implementation detail and that the fact `rock_gtest` and `rock_testsuite` handle this internally was by design (the latter even does find_package(Boost ...) internally as well). Feels like an anti-pattern and adds unnecessary boilerplate. Is there a reason for this?

Reply:
`rock_testsuite` and `rock_gtest` do not handle `ROCK_TEST_ENABLED`. `rock_standard_layout` does.

This is the separation of intent and options. I've unfortunately lost it, but someone recently wrote an article about writing modern CMake code that resonated with me: don't try to guess the intent. If `rock_gtest` is called, I mean to setup a GTest suite. 

In this particular case, our options are even more limited since we need to `find_package(Syskit)` to get the macro, which is not part of the widely present and low-on-dependencies `base/cmake`, but the heavy-on-dependency `syskit` - which we sometimes don't want to install on e.g. embedded targets.